### PR TITLE
Removed description of hazelcast-member tool.

### PR DESCRIPTION
**Pull request by stalek71**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -185,83 +184,6 @@ The following is an example configuration when you are adding an `IdentifiedData
...
-
-To stop a member, run the following command:
-
-```
-hazelcast-member stop
-```
-
-You can find more information about the `hazelcast-member` tool at its GitHub [repo](https://github.com/hazelcast/hazelcast-member-tool).
-
-See the [Hazelcast IMDG Reference Manual](http://docs.hazelcast.org/docs/latest/manual/html-single/index.html#getting-started) for more information on setting up the clusters.
```
Comment (ihsandemir) [inline README.md]: I left the last sentence for C++ README since it was not really specific to member tool.

Comment (Serdaro) [reply README.md]: Same applies here I guess?

Comment (stalek71) [reply README.md]: Yes, I will recover this fragment in a moment

**Comment by stalek71** [comment]
verify

**Comment by stalek71** [comment]
verify

**Comment by stalek71** [comment]
verify

**Pull request merged**